src/components/Common/Pill.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+import TokenList from 'decentraland-gatsby/dist/utils/dom/TokenList'
+
+import './Pill.css'
+
+export enum PillColor {
+  Green = 'green',
+  Blue = 'blue',
+  Red = 'red',
+  Gray = 'gray',
+  Purple = 'purple',
+  Fuchsia = 'fuchsia',
+  Orange = 'orange',
+  Yellow = 'yellow',
+}
+
+export type PillColors = 'green' | 'blue' | 'red' | 'gray' | 'purple' | 'fuchsia' | 'orange' | 'yellow'
```

Comment:
This is great! Good to know. 

I prefer to use PillColors in this case as it is later exported and used in other components, to avoid renaming or using just "Colors" without context.